samcli/commands/local/cli_common/invoke_context.py

```diff
@@ -270,6 +271,15 @@ def __enter__(self) -> "InvokeContext":
         if self._containers_initializing_mode == ContainersInitializationMode.EAGER:
             self._initialize_all_functions_containers()
 
+        for func in self._function_provider.get_all():
+            if func.packagetype == ZIP and func.inlinecode:
+                LOG.info(
```

Comment:
Shouldn't this be `LOG.warning`?

Reply:
yeah LOG.warn would make sense here, I can change that.

Reply:
Noticed LOG.warn is a deprecated method, changed it to LOG.warning just like your suggestion.